consensus.go

```diff
@@ -683,16 +683,14 @@ func (p *Pbft) forceTimeout() {
 	p.forceTimeoutCh = true
 }
 
-// randomTimeout calculates the timeout duration depending on the current round
-func (p *Pbft) randomTimeout(inputTimeout ...time.Duration) time.Duration {
-	timeout := 2 * time.Second
-	if len(inputTimeout) == 1 {
-		timeout = inputTimeout[0]
-	}
-
+// exponentialTimeout calculates the timeout duration depending on the current round
+func (p *Pbft) exponentialTimeout() time.Duration {
+	timeout := defaultTimeout
 	round := p.state.view.Round
-	if round > 0 {
-		timeout += time.Duration(math.Pow(2, float64(round))) * time.Second
+	if round <= 8 {
```

Comment:
Can you use `maxTimeout` here?

Reply:
Added a comment here about for <=8 as discussed.